app/assets/javascripts/components/util/routes.jsx

```diff
@@ -23,15 +21,13 @@ const routes = (
     <Route path="/courses/:course_school/:course_title" component={Course} />
     <Route path="/course_creator" component={ConnectedCourseCreator} />
     <Route path="/users/:username" component={UserProfile} />
-    <Route path="/campaigns/:campaign_slug/overview" component={CampaignOverviewHandler} />
```

Comment:
Leave this overview route in place, as it is needed for rendering the 'Download stats' button and we don't yet have the new React version of the overview ready.